Tasks/UsePythonVersionV0/Tests/L0.ts

```diff
@@ -22,7 +22,9 @@ describe('UsePythonVersion L0 Suite', function () {
         require('./L0_versionspec');
     });
 
-    it('succeeds when version is found', function () {
+    it('succeeds when version is found', function (done: Mocha.Done) {
+        this.timeout(parseInt(process.env.TASK_TEST_TIMEOUT as string) || 5000);
```

Comment:
Do we need to insert timeout for every test? Could it be enough to mention it in describe?

Reply:
Thanks! Let me try this.

Reply:
Applied change for this.